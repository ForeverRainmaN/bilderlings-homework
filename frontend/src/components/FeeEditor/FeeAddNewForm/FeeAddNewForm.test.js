import React from 'react';
import ShallowRenderer from 'react-test-renderer';
import AddFeeForm from "./FeeAddNewForm";

it('tests if Add fee form renders correctly', () => {
  const addFeeForm = ShallowRenderer
  .create(<AddFeeForm
      addFee={() => {
      }}
  />).toJSON();
  expect(addFeeForm).toMatchSnapshot();
});