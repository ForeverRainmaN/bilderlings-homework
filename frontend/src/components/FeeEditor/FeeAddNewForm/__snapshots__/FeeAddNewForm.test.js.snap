// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests if AddFeeForm renders correctly 1`] = `
<div
  className="bilderlings-homework-add-fee-form display-flex"
>
  <div
    className="bilderlings-homework-add-fee-form-selects display-flex"
  >
    <select
      className="bliderlings-homework-select-from width70-br5-mr-30"
      defaultValue="USD"
      id="from"
      onChange={[Function]}
    >
      <option>
        USD
      </option>
      <option>
        EUR
      </option>
      <option>
        RUB
      </option>
    </select>
    <select
      className="bliderlings-homework-select-to width70-br5-mr-30"
      defaultValue="RUB"
      id="to"
      onChange={[Function]}
    >
      <option>
        RUB
      </option>
      <option>
        USD
      </option>
      <option>
        EUR
      </option>
    </select>
  </div>
  <div
    className="bilderlings-homework-amount-input width70-br5-mr-30"
  >
    <div
      className="bp3-input-group"
    >
      <input
        autoComplete="off"
        className="bp3-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyPress={[Function]}
        onPaste={[Function]}
        style={
          Object {
            "paddingRight": 10,
          }
        }
        type="text"
        value="0"
      />
    </div>
  </div>
  <button
    className="bp3-button bp3-intent-primary width70-br5-mr-30"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    type="button"
  >
    <span
      className="bp3-button-text"
    >
      Add
    </span>
  </button>
</div>
`;
